File: OCH_Tao/ABC/B/243.py
N = int(input())
A = list(map(int,input().split()))
B = list(map(int,input().split()))
cnt = 0
X = set()
Y = set()
for i in range(N):
  if A[i]==B[i]:
    cnt+=1
  else:
    X.add(A[i])
    Y.add(B[i])
print(cnt)
print(len(X&Y))

File: OCH_Tao/ABC/B/243.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const N = parseInt(input[0], 10);
const A = input[1].split(" ");
const B = input[2].split(" ");
let cnt = 0;
let X = new Set();
let Y = new Set();
for (let i = 0; i < N; i++) {
  if (A[i] === B[i]) {
    cnt++;
  } else {
    X.add(A[i]);
    Y.add(B[i]);
  }
}
console.log(cnt);
let ans = new Set();
for (const i of X) {
  if (Y.has(i)) {
    ans.add(i);
  }
}
console.log(ans.size);